build/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ "${1:0:1}" = '-' ]; then
+    set -- mattermost "$@"
+fi
+
+exec "$@"
```

Comment:
No newline at the end of the file, I guess it's not on purpose.

Reply:
done